source/manual/deploying.html.md

```diff
@@ -17,27 +17,44 @@ If you are responding to a security incident, follow the steps to [deploy fixes
 [reg]: https://gds.blog.gov.uk/2012/11/02/regular-releases-reduce-risk
 [sec]: /manual/deploy-fixes-for-a-security-vulnerability.html
 
-## 1. Test on integration
+## Deploying a branch
+
+It can be useful to test your branch in a non-local environment before it is merged to master. Note that you should only do this on Integration, unless you have a good reason to deploy to a different environment.
```

Comment:
We could also put a warning that merging to master will automatically deploy to integration anyway, and if you've got a branch deployed this could get overwritten at any time.

Reply:
Have now added a warning just after the opening paragraph of this section 👍